getsong/getsong.py

```diff
@@ -37,18 +38,14 @@ def main():
     if not uri:
         print("Could not find result for {}. http://youtube.com/results?{}".format(term, urlencode({'search_query': term})))
         return 2
+    response = input("About to download the audio for {}.\nIs this correct? [y/n] ".format(title))
     try:
-        input = raw_input
-    except NameError:
-        pass
-    try:
-        cont = args.yes or input("About to download the audio for {}, is this correct? [y] ".format(title)) or "y"
+        cont = args.yes or response in ['Y', 'y']
```

Comment:
Might want to change this to `cont = args.yes or response in ['Y', 'y', 'Yes', 'yes']`